src/axom/slic/interface/slic.cpp

```diff
@@ -18,11 +18,23 @@ namespace slic
 //------------------------------------------------------------------------------
 bool debug::checksAreErrors = false;
 
-void initialize() { Logger::initialize(); }
+static bool root = true;
+
+void initialize(bool rootValue)
+{
+  axom::slic::root = rootValue;
+  Logger::initialize();
+}
 
 //------------------------------------------------------------------------------
 bool isInitialized() { return (Logger::getActiveLogger() != nullptr); }
 
+//------------------------------------------------------------------------------
+bool isRoot() { return axom::slic::root; }
+
+//------------------------------------------------------------------------------
+void setRoot(bool rootValue) { axom::slic::root = rootValue; }
```

Comment:
Suggestion -- to highlight that this is a bool:
```suggestion
void setIsRoot(bool rootValue) { axom::slic::root = rootValue; }
```